tests/test_finder.py

```diff
@@ -57,6 +57,40 @@ def calculate_chunk(self, file, start_offset: int):
         raise Exception("Error")
 
 
+class TestHandlerForward(Handler):
+    """Matches stuff between parens: (whatever)
+
+    Matches the initial ( and finds the next ) as chunk end.
+    """
+
+    NAME = "handlerEXC"
+    PATTERNS = [Regex("[(]")]
+
+    def calculate_chunk(self, file, start_offset: int):
+        end_offset = file.find(b")", start_offset)
+        assert end_offset >= 0
+        return ValidChunk(start_offset, end_offset + 1)
+
+
+class TestHandlerBackward(Handler):
+    """Matches stuff between square brackets: [whatever]
+
+    Matches the closing ] and finds the previous [ as chunk start.
+
+    This is not a common strategy, but can be a valid/more efficient strategy in some cases
```

Comment:
While this could be a valid strategy, this would fundamentally change how we think about chunk matching. So far all handlers are working in a start->end direction and we use this to optimize our chunk finding. Changing it would have a severe performance impact on how unblob works.